.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Install yarn
+        run: sudo apt update && sudo apt install yarn
+
+      - name: Install Dependencies
+        run: yarn install
+
+      - name: Jest Unit Test & Linting
+        run: yarn test
+
+      - name: Build
+        run: yarn build
+
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@master
+        id: semantic
+        with:
+          branch: master
+          extra_plugins: |
+            @semantic-release/git
+            @semantic-release/changelog
+
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN_KEY }}
```

Comment:
`secrets .GITHUB_TOKEN` is automatically available in workflows

Reply:
Thanks for the review @jquense, are you recomending removing this env or replacing with `secrets.GITHUB_TOKEN`?

Reply:
I think replacing with `secrets.GITHUB_TOKEN`

Reply:
This is done, I found the doc in github 

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token

FYI: I also updated the npm token key value to `NPM_TOKEN`